contrib/jenkins/pyfmt.groovy

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/groovy
+
+/*
+ * Install and run pyfmt
+ *
+ * code_dirs is a string representing the paths (space delimted) to pass to pyfmt
+ */
+def call(String code_dirs) {
+    sh "pip3 install --quiet black isort"
```

Comment:
i'm not sure how often these projects get updated but builds/pushes may unexpectedly fail in the future if the checkers implement some new features and the jenkins pipeline command pulls new versions during each build.

Reply:
good point. i can pin versions